### PR TITLE
Remove to-do button

### PR DESCRIPTION
**Pull request by ebilg**
Remove button

### CODE REVIEW

File: src/components/TaskCards.js [hunk 1]
```diff
@@ -96,7 +96,6 @@ const AddToDoLink = () => {
   return (
     <>
       <AddToDoModal open={openModal} close={handleCloseModal} />
```
Comment (achidu) [inline src/components/TaskCards.js:98]: It _might_ make sense to remove this reference to the modal here as well

Comment (achidu) [inline src/components/TaskCards.js:98]: It _might_ make sense to remove this reference to the modal here as well

Comment (ebilg) [reply src/components/TaskCards.js]: You're right, I think it's better to comment out all of this!

Comment (ezekielp) [reply src/components/TaskCards.js]: 👋  Any follow-up on this? I know you're busy @ebilg , just wanted to check in

Comment (achidu) [reply src/components/TaskCards.js]: @ezekielp i _believe_ those changes were made in the last commit "comment modal code"

Comment (ebilg) [reply src/components/TaskCards.js]: yes sorry, this is ready for re-review now :)

**Review by ezekielp — APPROVED** [review]

**Pull request merged**